libpod/container_internal_linux.go

```diff
@@ -712,7 +742,7 @@ func (c *Container) makeBindMounts() error {
 			}
 			c.state.BindMounts["/etc/resolv.conf"] = newResolv
 
-			newHosts, err := c.generateHosts()
+			newHosts, err := c.generateHosts("/etc/hosts", c)
```

Comment:
I'd use defaultHostFile here too

Reply:
aw shoot that was the point of adding it how did I miss it lol